sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/SqlOperators.java

```diff
@@ -82,58 +83,85 @@
           new UdafImpl<>(new StringAgg.StringAggString()));
 
   public static final SqlOperator START_WITHS =
-      createUdfOperator("STARTS_WITH", BeamBuiltinMethods.STARTS_WITH_METHOD);
+      createUdfOperator(
```

Comment:
for these `ZETASQL_FUNCTION_GROUP_NAME`, eventually I think we will migrate it off from `createUdfOperator`.